src/workflows/airqo_etl_utils/data_validator.py

```
@@ -159,12 +159,12 @@ def remove_outliers(data: pd.DataFrame) -> pd.DataFrame:
                         column_name=mapped_name, row_value=x
                     )
                 )
-        else:
-            data[col] = data[col].apply(
-                lambda x: DataValidationUtils.get_valid_value(
-                    column_name=mapped_name, row_value=x
+            else:
+                data[col] = data[col].apply(
+                    lambda x: DataValidationUtils.get_valid_value(
+                        column_name=mapped_name, row_value=x
```

Ruff (0.8.0)

Function definition does not bind loop variable `mapped_name`

(B023)